lib/glific/partners.ex

```diff
@@ -816,7 +816,7 @@ defmodule Glific.Partners do
   """
 
   @spec update_credential(Credential.t(), map()) ::
-          {:ok, Credential.t()} | {:error, Ecto.Changeset.t()}
+          {:ok, any} | {:error, any}
```

Comment:
I think it still return {:ok, Credential.t()}

Reply:
done